deltachat-ffi/deltachat.h

```diff
@@ -1037,14 +1037,18 @@ int dc_send_webxdc_status_update (dc_context_t* context, uint32_t msg_id, const
  * @memberof dc_context_t
  * @param context The context object
  * @param msg_id id of the message with the webxdc instance
- * @param status_update_id Can be used to filter out only a concrete status update.
- *     When set to 0, all known status updates are returned.
- * @return JSON-array containing the requested updates,
- *     each element was created by dc_send_webxdc_status_update()
- *     on this or other devices.
- *     If there are no updates, an empty JSON-array is returned.
+ * @param serial The last known serial, defaults to 0.
+ * @return JSON-array containing the requested updates.
+ *     Each `update` comes with the following properties:
+ *     - `update.payload`: equals the payload given to dc_send_webxdc_status_update()
+ *     - `update.serial`: the serial number of this update. The first update will have the serial `1`.
+ *     - `update.max_serial`: the maximum serial currently known.
```

Comment:
This `max_serial` is the same for all updates returned?

Reply:
yes.